kuka_rsi_hw_interface/CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(kuka_rsi_hw_interface)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
```

Comment:
If we're not going to do this, let's just use:

```cmake
set(CMAKE_CXX_STANDARD 11)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Don't rely on environmental factors.

Reply:
this requires at least CMake 3.1 (minimum recommended requirement for melodic is 3.7.2 so that would be fine)